client/src/actions/types.ts

```diff
@@ -0,0 +1,10 @@
+export const LOGIN_SUCCESS: string = "LOGIN_SUCCESS";
+export const LOGIN_FAILURE: string = "LOGIN_FAILURE";
+
+export const REGISTER_SUCCESS: string = "REGISTER_SUCCESS";
+export const REGISTER_FAILURE: string = "REGISTER_FAILURE";
+
+export const USER_LOADED: string = "USER_LOADED";
+export const AUTH_ERROR: string = "AUTH_ERROR";
+
+export const LOG_OUT: string = "LOG_OUT";
```

Comment:
Create an object, that will contain all of these types

Reply:
done